systemtest/src/main/java/io/strimzi/systemtest/TestTags.java

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest;
+
+/**
+ * Defines constants used as tags in the tests. These tags categorize and filter
+ * tests based on their purpose, stability, and environment, enabling selective execution of tests
+ * for scenarios like regression, scalability, upgrades, and performance.
+ */
+public interface TestTags {
+
+    /**
+     * Feature gate related constants
+     */
+    // No Feature gates kept for STs at this moment
+
+    /**
+     * Default value which allows execution of tests with any tags
+     */
+    String DEFAULT_TAG = "all";
```

Comment:
Is this needed? I mean, we are not tagging anything with this. We are just specifying `-P all` in case of need (which is not tag, but profile name)

Reply:
Sure, I can remove it.